packages/accounts-base/accounts_server.js

```diff
@@ -176,9 +176,10 @@ Ap._failedLogin = function (connection, attempt) {
   });
 };
 
-Ap._successfulLogout = function () {
+Ap._successfulLogout = function (connection, userId) {
+  const user = userId && this.users.findOne(userId);
```

Comment:
I do not see `user` to be passed to `onLogin`? `connection` is cheap, because it is already available. But `user` object is not.